### PR TITLE
Fix OCP-9639

### PR DESCRIPTION
**Pull request by liangxia**
Limit PV/PVC in storage class so it does not interfere with other test cases.

@chao007 @duanwei33

### CODE REVIEW

File: features/storage/hostpath.feature [hunk 1]
```diff
@@ -10,12 +10,14 @@ Feature: Storage of Hostpath plugin testing
       | ["spec"]["hostPath"]["path"] | /etc/origin/hostpath/<%= project.name %> |
       | ["spec"]["hostPath"]["type"] | DirectoryOrCreate                        |
       | ["spec"]["accessModes"][0]   | <access_mode>                            |
+      | ["spec"]["storageClassName"] | sc-<%= project.name %>                   |
       | ["spec"]["persistentVolumeReclaimPolicy"] | <reclaim_policy>            |
```
Comment (chao007) [inline features/storage/hostpath.feature]: There is not align

Comment (pruan-rht) [reply features/storage/hostpath.feature]: @chao007 hmmm...alignment looks fine.

Comment (liangxia) [reply features/storage/hostpath.feature]: @chao007 The alignment issue on line 14 is not related to this PR. Anyway, i will fix it.

Comment (liangxia) [reply features/storage/hostpath.feature]: Now the alignment issue is fixed. The real change of adding storageClassName is not so obviously now.

**Comment by liangxia** [comment]
@chao007 @duanwei33  PTAL

**Pull request merged**